src/utils/DependencyUtils.ts

```diff
@@ -11,3 +29,62 @@ export const splitToNameAndVersion = (dependencyString: string): [string, string
     const fullName = parts.join('-');
     return [fullName, version];
 };
+
+export async function getFullDependencyList(
+    combos: ThunderstoreCombo[],
+    game: Game,
+    installedMods: ManifestV2[],
+    modPackDependencySetBuilderMode: ModPackDependencySetBuilderMode = ModPackDependencySetBuilderMode.USE_EXACT_VERSION
+) {
+    const originalCombos = [...combos];
+    for (const combo of originalCombos) {
+        const isModpack = combo.getMod().getCategories().some(value => value === "Modpacks");
+        const versionMode = isModpack ? modPackDependencySetBuilderMode : ModPackDependencySetBuilderMode.USE_LATEST_VERSION;
+
+        await buildDependencySet(combo.getVersion(), game, combos, versionMode);
+
+        // Only filter if this combo is not a modpack and there are some installed mods
```

Comment:
Yes, the next line tells me as much. What this comment should tell me is WHY we want this behaviour, especially WHY mods and modpacks are treated differently.

Reply:
This filtering is now explained in the big beautiful comment ™️  above the function.